jupyterhub_config.py

```diff
@@ -0,0 +1,13 @@
+"""
+sample jupyterhub config file for testing tmpauthenticator
+"""
+
+c = get_config()  # noqa
+
+from tmpauthenticator import TmpAuthenticator
+
+c.JupyterHub.authenticator_class = TmpAuthenticator
```

Comment:
Either works, but this pattern is easier for users sometimes, for example configuring this via YAML `hub.config.JupyterHub.authenticator_class` in z2jh.

```suggestion
c.JupyterHub.authenticator_class = "tmp"
```

- Possible since #38